test/e2e/utils_test.go

```diff
@@ -81,6 +83,19 @@ func getKubeClient(kubeConfigPath string) (*kubernetes.Clientset, error) {
 	return clientset, nil
 }
 
+// getKubeConfig creates a REST config from the kubeconfig file.
+func getKubeConfig() (*rest.Config, error) {
```

Comment:
I think this helper can be reused in `test/e2e/e2e_test.go` `beforeSuiteAllProcesses()` and `utils_test.go` `getKubeClient()`? They use very similar code.